src/Microsoft.PowerShell.Commands.Utility/commands/utility/ImplicitRemotingCommands.cs

```diff
@@ -2727,7 +2727,14 @@ private string GenerateCertificateThumbprintParameter()
 
```

Comment:
@KirkMunro, your last commit had 1 failures in `PowerShell-CI-windows`
Tests for (error, warning, etc) action preference.action preference of Suspend cannot be set as a preference variable
```powershell
Expected an exception, with FullyQualifiedErrorId 'RuntimeException' to be thrown, but no exception was thrown.
at <ScriptBlock>, D:\a\1\s\test\powershell\Language\Scripting\ActionPreference.Tests.ps1: line 66
66:             } | Should -Throw -ErrorId 'RuntimeException' -PassThru
```